### ISSUE TITLE
Custom checkboxes and radios retheming

### ISSUE THREAD

**Issue by MartijnCuppens** [twbs/bootstrap#27063]
The custom checkboxes and radios are cool, but I think they might need some retheming. I don't think it's a good idea to make them gray, it looks like they are disabled and there isn't much of a difference between the disabled and not disabled versions.

Imo they should have the same background and border color as the `.form-control` elements and also take over the theming of disabled state.

I made a demo of the rethemed custom checkboxes here: https://codepen.io/MartijnCuppens/pen/KBYKKz

**Comment by MartijnCuppens** [comment]
@andresgalante @XhmikosR @mdo, I want to check if we can optimise the css of the custom form elements, but it would be nice if we can decide whether or not we're going to continue with the retheming I suggested here to avoid merge conflicts.

**Comment by XhmikosR** [comment]
Yup, I personally totally agree with you that the current custom background doesn't look good.

On the other hand, I'd go with a little darker border color than your pen :)

**Comment by MartijnCuppens** [comment]
@XhmikosR, I've updated the PR and made a v2 with a darker border.
